conf/drupal/config/core.menu.static_menu_link_overrides.yml

```diff
@@ -5,5 +5,17 @@ definitions:
     parent: ''
     expanded: false
     weight: 0
+  user__page:
+    weight: -50
+    menu_name: account
+    parent: ''
+    enabled: true
+    expanded: false
+  user__logout:
+    weight: -49
+    menu_name: account
+    parent: ''
+    expanded: false
+    enabled: true
```

Comment:
I'd added a "get your ticket" link to the user menu a couple of days ago and it looks like it's not reflected here. Want to make sure that's not getting overridden.
![MidCamp_2021___March_24_-_27__2021](https://user-images.githubusercontent.com/238201/110965095-7f69e700-8319-11eb-9453-1e5b7c0161e7.png)